deploy-board/deploy_board/settings.py

```diff
@@ -83,6 +83,7 @@
 # Serviceframework add-on vars
 SERVICE_RATELIMIT_CONFIG_URL = os.getenv("SERVICE_RATELIMIT_CONFIG_URL")
 STATSBOARD_API_FORMAT = os.getenv("STATSBOARD_API_FORMAT", "OFF")
+STATSBOARD_API_PREFIX = os.getenv("STATSBOARD_API_PREFIX", None)
```

Comment:
```python
>>> url =''
>>> url.startswith(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: startswith first arg must be str or a tuple of str, not NoneType
```

Reply:
Yeah may not be great for the default to fail. Updated the default to `https://`